### ISSUE TITLE
Cookies are not persisted between requests

### ISSUE THREAD

**Issue by bartmoc** [restcli/restcli#47]
Currently, the cookie jar is not handled by the restcli. There is an option to disable it `isNoCookieJar` but it's never used. 
Is there a plan for handling persistent cookies between requests (cookie jar). This feature is available in IJ.

**Comment by quangson91** [comment]
Yes, I planned to implement the (cookie jar) in the next version.
I will update this issue when the code is done.
Thank you for the feedback!

**Comment by bartmoc** [comment]
Great news! Really good job on this project. Well done.
